recipes/full_finetune_distributed.py

```diff
@@ -291,7 +292,10 @@ def _setup_model(
         # across all available GPUs.
         model = FSDP(
             module=model,
-            auto_wrap_policy=ModuleWrapPolicy({modules.TransformerDecoderLayer}),
+            auto_wrap_policy=utils.get_full_finetune_fsdp_wrap_policy(
+                memory_efficient_wrapping=cfg.memory_efficient_wrapping,
```

Comment:
```suggestion
                memory_efficient_wrapping=memory_efficient_wrapping,
```

Reply:
yep, thanks for catching these! Accidentally committed a wrong version of the code.